faq.md

```diff
@@ -78,7 +78,7 @@ available, but also supports other mechanisms.
 
 Apart from the medik8s team being actively involved with the [Machine
 Healthcheck
-Controller](https://github.com/kubernetes-sigs/cluster-api/blob/master/controllers/machinehealthcheck_controller.go),
+Controller](https://github.com/kubernetes-sigs/cluster-api/blob/main/controllers/machinehealthcheck_controller.go),
```

Comment:
Looking at it again, there is no `machinehealthcheck_controller.go` under `https://github.com/kubernetes-sigs/cluster-api/blob/main/controllers`. Can it be outdated and there is no such file anymore?

Reply:
https://github.com/kubernetes-sigs/cluster-api/blob/main/internal/controllers/machinehealthcheck/machinehealthcheck_controller.go